vt-digital-libraries-platform/resolution-service
# update VTDLP Resolution Service to using SAM and one-click button

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2281) (:star:)

# What does this Pull Request do? (:star:)
Update VTDLP Resolution Service to using SAM and one-click button

# How should this be tested?
1. Read the updated readme and do one-click or Sam deploy
2. Verify the service is working

# Additional Notes:
You can update the exiting cloudformation stack if you use current master branch to deploy.

# Interested parties
Tag (@vt-digital-libraries-platform/diglib-dev ) interested parties

(:star:) Required fields
